til-api/src/main/java/com/til/controller/problem/reponse/ProblemListResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.til.controller.problem.reponse;
+
+import java.util.List;
+
+import com.til.domain.problem.dto.ProblemInfoDto;
+
+public record ProblemListResponse(
+                                  List<ProblemInfoDto> problemInfoDtoList
+) {
+
+    public static ProblemListResponse of(List<ProblemInfoDto> problemInfoDtoList) {
+        return new ProblemListResponse(problemInfoDtoList);
```

Comment:
```suggestion
public record ProblemInfoListResponse(
                                  List<ProblemInfoDto> problemInfoDtoList
) {

    public static ProblemInfoListResponse of(List<ProblemInfoDto> problemInfoDtoList) {
        return new ProblemInfoListResponse(problemInfoDtoList);
```

P3: 엔티티명인 `ProblemInfo` DTO명인 `ProblemInfoDto` 와 일관성을 맞추기 위해 `ProblemInfoListResponse`로 이름을 변경하는건 어떠신가요?
해당 이름이 너무 길면 `Info` 라는 단어를 제거하고 `Problem` `ProblemDto` `ProblemListResponse` 로 사용하는 방법도 괜찮을 것 같습니다!

Reply:
저도 이 부분을 어떻게 할지 고민했는데..
`info`를 빼고 `Problem` `ProblemDto` `ProblemListResponse` 이렇게 가는게 더 좋을 것 같습니다. 
이렇게 수정하겠습니다!

Reply:
controller 단에 있는 request나, response는 뒤에 네이밍을 ~Requset, ~Response로 통일시키면 좋을 것 같습니다😀